docs/book/src/cronjob-tutorial/testdata/project/config/policy/allow-metrics-traffic.yaml

```diff
@@ -0,0 +1,29 @@
+# This NetworkPolicy allows ingress and egress traffic
+# with Pods running on namespaces labeled with 'metrics: enabled'. Only Pods on those
+# namespaces are able to gathering data from the metrics endpoint.
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  labels:
+    app.kubernetes.io/name: project
+    app.kubernetes.io/managed-by: kustomize
+  name: allow-metrics-traffic
+  namespace: system
+spec:
+  podSelector:
+    matchLabels:
+      control-plane: controller-manager
+  policyTypes:
+    - Egress
```

Comment:
Why are we addressing egress traffic in these network policies at all? I don't think egress traffic is required in general from operators, except that it must be able to reach the API server. Access to the API server should be addressed more generally if using network policies in a cluster.

As a cluster-admin I would not be happy if a workload is granted any egress traffic if not required.